lib/global_id/uri/gid.rb

```diff
@@ -83,7 +83,7 @@ def create(app, model, params = nil)
       def build(args)
         parts = Util.make_components_hash(self, args)
         parts[:host] = parts[:app]
-        parts[:path] = "/#{parts[:model_name]}/#{parts[:model_id]}"
+        parts[:path] = "/#{parts[:model_name]}/#{CGI.escape(parts[:model_id].to_s)}"
```

Comment:
We could. But I should we support that? I feel like we're ballooning possible values to have any format, and I guess I like the constraints of the gid scheme.

@jeremy do you have a preference here?